src/util/postprocess_maf.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+from util.util import gsutil_cp
+
+def download_maf_from_workspace(refwm, sample_set_ids = ['all_ice', 'all_agilent'],
```

Comment:
it is a one liner usually. the merging tool that you call before should have done this step already. the thing is you calleed it on each sample set instead of calling it on the "All" sample set.